source/frequently-asked-questions/frequently-asked-questions.rst

```diff
@@ -2,3 +2,21 @@
 Frequently Asked Questions
 ################################################################################
 
+* **How can I store big files on the blockchain?**
+  Swarm_ is an Ethereum-specific project for distributed file storage. IPFS_ is an independent project which has close ties to Ethereum; it will be used independently and may be used as the layer underlying Swarm.
+* **Is ethereum based on bitcoin?**
+  Only in the sense that it uses a blockchain, which Bitcoin pioneered. Ethereum has a separate blockchain that has several signigicant technical differences from Bitcoin. 
+* **What other cool apps are being built?** See this list from Ethercasts_.
+* **What's the future of Ethereum?** We are planning a switch to Proof of Stake_ in the near future. We are also investigating scalability_ solutions and how to store_ secrets on the blockchain
```

Comment:
Best to refer to the roadmap section. 
There will be a section on PoS and scalability with links, best refer to that